ticket.go

```diff
@@ -59,13 +59,20 @@ func OpenStore(r io.Reader) (*Store, error) {
 	return s, nil
 }
 
-// Yada.
+// WriteTo writes encodes data to JSON and writes to file.
 func (s *Store) WriteTo(w io.Writer) error {
```

Comment:
It strikes me that maybe a better name for this is `WriteJSONTo`—what do you think? Then we avoid the problem that we're not implementing the standard `WriteTo` method, which returns `(int, error)`.